test/lib/input/config.test.ts

```diff
@@ -0,0 +1,99 @@
+import {SfCommand} from '@salesforce/sf-plugins-core';
+import {expect} from 'chai';
+import {CONFIG_FILE, flags as configFlags, ConfigInput} from '../../../src/lib/input/config';
+
+describe('config.ts', () => {
+	describe('flags', () => {
+		// Declare a dummy Command that we can use to validate the functionality of the flags.
+		class TestCommand extends SfCommand<string> {
+			public static readonly flags = {
+				...configFlags
+			};
+
+			public async run(): Promise<string> {
+				const parsedFlags: ConfigInput = (await this.parse(TestCommand)).flags;
+				return parsedFlags[CONFIG_FILE];
+			}
+		}
+
+		describe('-c/--config-file', () => {
+			describe('Long name: --config-file', () => {
+				it('Rejects non-existent file', async () => {
+					const inputValue = 'definitely-not-a-real-file.txt';
+					let errorThrown: boolean;
+					let message: string;
+					try {
+						await TestCommand.run(['--config-file', inputValue]);
+						errorThrown = false;
+					} catch (e) {
+						errorThrown = true;
+						message = e.message;
+					}
+					expect(errorThrown).to.equal(true, 'Error should be thrown');
+					expect(message).to.contain(`No file found at ${inputValue}`);
+				});
+
+				it('Accepts existent file', async () => {
```

Comment:
Just a recommendation. One useful way of writing tests so that they read as locked in requirements is to have each test name read as "When ..., then ...". Not saying you have to go with this pattern, but it does help me as a reviewer of your tests know what requirement is getting locked in a little better.